src/services/notifications/index.ts

```diff
@@ -191,13 +194,59 @@ export const NotificationsService = (logger: Logger): INotificationsService => {
     }
   }
 
+  const sendBalance = async (account: Account): Promise<void> => {
+    const balanceSats = await getBalanceForWallet({
+      walletId: account.defaultWalletId,
+      logger,
+    })
+    if (balanceSats instanceof Error) throw balanceSats
+
+    // Add commas to balancesats
+    const balanceSatsPrettified = balanceSats.toLocaleString("en")
+    // Round balanceusd to 2 decimal places and add commas
+
+    let balanceUsd: string, title: string
+    const price = await getCurrentPrice()
+    if (price instanceof Error) {
+      logger.warn({ price }, "impossible to fetch price for notification")
+
+      // TODO: i18n
+      title = `Your balance is ${balanceSatsPrettified} sats)`
+    } else {
+      const usdValue = price * balanceSats
+      balanceUsd = usdValue.toLocaleString("en", {
+        maximumFractionDigits: 2,
+      })
+
+      // TODO: i18n
+      title = `Your balance is $${balanceUsd} (${balanceSatsPrettified} sats)`
+    }
+
+    logger.info(
+      { balanceSatsPrettified, title, ownerId: account.ownerId },
+      `sending balance notification to user`,
+    )
+
+    // FIXME:
```

Comment:
What needs to be fixed here?

Reply:
the line below. we should not pass a `UserType` to `sendNotification`. but this requires more refactoring of `sendNotification` (not part of what this PR do)